### DISCUSSION TITLE
Normal to completely re-index every start? (0.15.0-6b12a45)

### DISCUSSION THREAD

**Discussion by matknowles** [blakeblackshear/frigate#15577] [Ask A Question] [answered]
Hey everyone, on 0.15.0-6b12a45 is it normal to completely reindex the semantic search on every start? I'm not sure if it's something to do with my database/storage being on a remote NAS, or just so many tracked objects overlooking a busy road. It's taking a very long time to get to the search screen despite having hardware acceleration working. It doesn't seem to be just reindexing new objects, or just objects without descriptions.

![General_Stats_-_Frigate_and_1_more_page_-_Personal Screenshot 2024 12 18 - 14 49 35](https://github.com/user-attachments/assets/28b02e31-6479-4592-a65c-c4124d65d8ca)

(this has been going for roughly an hour already)

I've been iterating some configs recently and it's a bit of a pain not being able to use the search for so long after every start.

I'm using docker on linux and an NFS NAS share mounted for config and media, with an entry in /etc/fstab to mount on boot.

I'm not complaining, just wondering if my config/setup might be faulty. The feature is legendary by the way, amazing work.

**Comment by NickM-27** [comment, marked as answer]
you need to remove 

```
reindex: true
```

from your semantic_search config

**Reply by matknowles** [reply]
Oh dear, I missed it. Sorry about that.

**Comment by hawkeye217** [comment]
Docs are helpful here, see the tip: https://deploy-preview-13787--frigate-docs.netlify.app/configuration/semantic_search#configuration 

> Make sure to turn the UI's switch off or set the config back to False before restarting Frigate again.

You can either edit your config yourself or use the switch in the UI. The way I do it is to enable the "Re-Index on Startup" switch in Search Settings, Save, restart, and then immediately go to Settings and flip the Reindex switch off. Easy.

**Reply by matknowles** [reply]
The documentation has changed since I set it up to be fair, but it's my bad all the same. Thanks again.
